Sources/LiveViewNativeCore/LiveViewNative.swift

```diff
@@ -259,6 +304,15 @@ extension Attribute: Hashable {
 public struct AttributeName {
     public var namespace: String?
     public var name: String
+    
+    /// The textual representation (`namespace:name` if it has a namespace, otherwise just the name) of this attribute name.
+    public var value: String {
```

Comment:
"the name as a value? what?"

Yeah, this is not a good name for this property. If you have a better idea, I'm all ears

Reply:
Can't the Identifiable implementation be used for this purpose? Feels redundant to me. If not that, if Swift has the equivalent of the Rust `ToString` trait, that'd probably be preferable versus a dedicated property/field for getting the fully-qualified name as a string. Thoughts?

Reply:
It could (or they could all delegate to the same implementation). The Swift equivalent is the `CustomStringConvertible` protocol, but its `description` also feels like a weird name.

I think the `RawRepresentable` protocol is the best option, with a string `rawValue` property. The `CustomStringConvertible` and `Identifiable` implementations can then delegate to that.